packages/web-components/fast-components-msft/src/tree-view/fixtures/tree-view.html

```diff
@@ -2,33 +2,49 @@
     <h4>With items (don't render collapsed nodes)</h4>
     <fast-tree-view render-collapsed-nodes="false">
         <fast-tree-item>
-            Root item
-            <fast-tree-item slot="item">
+            Root item 1
+            <fast-divider slot="item"></fast-divider>
```

Comment:
I thought we were going to try to avoid declaring things that are not items with the slot "item"?